modules/appeals_api/app/swagger/appeals_api/v2/swagger_dev.json

```diff
@@ -10345,7 +10347,7 @@
           "Supplemental Claims"
         ],
         "operationId": "postSupplementalClaimEvidenceSubmission",
-        "description": "This is the first step to submitting supporting evidence for an SC.  (See the Evidence Uploads section above for additional information.)\nThe Supplemental Claim GUID that is returned when the SC is submitted, is supplied to this endpoint to ensure the SC is in a valid state for sending supporting evidence documents.\n",
+        "description": "This is the first step to submitting supporting evidence for an SC.  (See the Evidence Uploads section above for additional information.)\nThe Supplemental Claim GUID that is returned when the SC is submitted, is supplied to this endpoint to ensure the SC is in a valid state for sending supporting evidence documents.\n\nOnce a Supplemental Claim has been submitted, evidence may be uploaded up to 7 days from the Supplemental Claim's created_at date via 'supplemental_claims/evidence_submissions'.\n",
```

Comment:
Not a blocker, but there's a lot of mixing "Supplemental Claim" and "SC" in this. We should define the acronym in parenthesis at the first full mention, then use SC from then on.

e.g:
>...Supplemental Claims (SC)...The SC GUID this is returned when the SC is in a valid...Once a SC has been submitted...7 days from the SC's `created_at` date...